ingestors/npm.go

```diff
@@ -36,12 +35,21 @@ type NPMChangeDoc struct {
 	Time map[string]string `json:"time"`
 }
 
+func (ingestor *NPM) Name() string {
+	return "npm"
+}
+
 func (ingestor *NPM) Ingest(results chan data.PackageVersion) {
-	since, err := ingestor.GetLatestSequence()
+	since, err := getBookmark(ingestor, "now")
 	if err != nil {
 		log.WithFields(log.Fields{"ingestor": "npm"}).Fatal(err)
 	}
 
+	// TODO: This is a migration for bookmarks, can delete after first deployment
+	if since == nil {
```

Comment:
Will this ever trigger with the default value of now?

Reply:
It would have had we started here, but since it's already set, this is just a cheapo migration for us.

Reply:
Oh I see what you're saying, no it will not :D